specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -82,6 +82,61 @@
         }
       }
     },
+    "/managedidentitymodebackup": {
+      "post": {
+        "tags": [
+          "ManagedIdentityModeFullBackup"
+        ],
+        "operationId": "ManagedIdentityModeFullBackup",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
```

Comment:
You need to use data-plane LROs. This should be a status monitor per https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#obtaining-status-and-results-of-long-running-operations